modules/odr_core/odr_core/models/content.py

```diff
@@ -56,8 +76,22 @@ class ContentAuthor(Base):
     id = Column(Integer, primary_key=True, index=True)
     name = Column(String)
     url = Column(String, nullable=True)
-    content_id = Column(Integer, ForeignKey('contents.id'))
+    content_id = Column(Integer, ForeignKey("contents.id"))
     created_at = Column(DateTime(timezone=True), server_default=func.now())
     updated_at = Column(DateTime(timezone=True), onupdate=func.now())
 
     content = relationship("Content", back_populates="content_authors")
+
+
+class ContentSource(Base):
+    __tablename__ = "content_sources"
+
+    id = Column(Integer, primary_key=True, index=True)
+    content_id = Column(Integer, ForeignKey("contents.id"))
+    type = Column(Enum(ContentSourceType))
+    value = Column(String)
```

Comment:
We should make 'value' unique, as we don't expect to have multiple ContentSources with same address